src/parser/flex_lexer.l

```diff
@@ -211,6 +209,8 @@ ROLLBACK    TOKEN(ROLLBACK)
 COMMIT      TOKEN(COMMIT)
 INTERVAL    TOKEN(INTERVAL)
 
+CHARACTER[ \t]+VARYING TOKEN(CHARACTER_VARYING)
```

Comment:
I am totally fine with this version.
But normally, would we have to allow also newlines and comments between "CHARACTER" and "VARYING"?

Reply:
Good catch. I added newlines at least.